dynamics-nav/sendtracetag-function.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SENDTRACETAG Function"
+description: SENDTRACETAG Function Sends a telemetry trace event to the event log. This topic contains information on SENDTRACETAG Function syntax, parameters, remarks, and examples.
```

Comment:
Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.